src/Service/CacheManager.php

```diff
@@ -96,4 +96,16 @@ public function isCachingEnabled($filter, array $filterOptions)
 
         return $this->cacheOptions->getEnableCache();
     }
+    
+    /**
+     * {@inheritDoc}
+     */
+    public function useCacheUrl()
```

Comment:
Can you add this method to CacheManagerInterface with the appropriate docblocks?